lib/node_modules/@stdlib/math/base/special/fast/hypotf/docs/repl.txt

```diff
@@ -0,0 +1,33 @@
+
+{{alias}}( x, y )
+    Computes the hypotenuse of a single-precision floating-point number.
+
+    Parameters
+    ----------
+    x: number
+        First number.
+
+    y: number
+        Second number.
+
+    Returns
+    -------
+    out: number
+        Hypotenuse.
+
+    Examples
+    --------
+    > var h = {{alias}}( -5.0, 12.0 )
+    13.0
+
+    // For a sufficiently large `x` and/or `y`, the function overflows:
+    > h = {{alias}}( 1.0e154, 1.0e154 )
```

Comment:
Same thing here and below. The values need to be adjusted to within single-precision range.